keyboards/leafcutterlabs/bigknobv2/readme.md

```diff
@@ -0,0 +1,21 @@
+# bigknobv2
+
+![](https://i.imgur.com/8ZBC23k.jpg)
+![](https://i.imgur.com/I0IOA76.jpg)
+
+Macropad Media Controller with USB-C + RGB underglow + Hotswap and support for up to 40mm knob
+
+* Keyboard Maintainer: [Craig Gardner](https://github.com/leafcutterlabs)
+* Hardware Supported: bigknobv2
+* Hardware Availability: [Reddit](https://www.reddit.com/r/mechmarket/comments/gp8y3y/uscahbigknobv2_prototype_usbc_qmk_4_key_hotswap/)
+
+Notes:
+
+* Due to a hardware error, USB-C is not reversible (i.e., it only works in one orientation)
+* Due to `TAP_DANCE_ENABLE = yes`, the [QMK Configurator](https://config.qmk.fm/) will not be able to compile firmware (i.e., you must compile manually)
```

Comment:
This is only true if you enable Tap Dance at the keyboard level; Configurator's API ignores `rules.mk` files at the keymap level, so it will be able to compile firmware for this board (but without any tap dances or encoder functionality).

```suggestion
* Due to the use of Tap Dance, [QMK Configurator](https://config.qmk.fm/) will not be able to compile fully-featured firmware (i.e., you must compile manually).
```